app.py

```diff
@@ -30,18 +30,20 @@ def __repr__(self):
 dict = {}
 def parse_response(json):
     if 'text' in json['message']:
-        match = re.search(r'(\w+):(\w+):(.+)', json['message']['text'])
+        match = re.search(r'(\w+):(\.+):(.+)', json['message']['text'])
```

Comment:
Dit heeft niks te maken met de commit message. opsplitsen of niet doen. (alle tekens is . niet \.).